### PR TITLE
Improve distributed diployment doc

### PR DESCRIPTION
**Pull request by Krishanx92**
## Purpose
Fix https://github.com/wso2/docs-apim/issues/785

### CODE REVIEW

File: en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md [hunk 1]
```diff
@@ -91,3 +91,238 @@ In the following diagram, the five components are set up in a distributed deploy
...
+        <td>
+            Publishing events like block condition, token revocation
+        </td>
+    </tr>
+    <tr class="even">
+    <td>Publisher</td>
+    <td>Traffic Manager</td>
+    <td>9711</td>
+        <td>
+            authenticate to Publishing events
```
Comment (tgtshanika) [inline en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md:168]: Start with capital. Please fix this in other places as well.

Comment (Krishanx92) [reply en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md]: Fixed with https://github.com/wso2/docs-apim/pull/2794/commits/8c01a94ac09eb016f84e0d7213401826fe556d83

File: en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md [hunk 2]
```diff
@@ -91,3 +91,238 @@ In the following diagram, the five components are set up in a distributed deploy
...
+    <td>8099</td>
+        <td>
+            Gateway WS
+        </td>
+    </tr>
+    <tr class="odd">
+    <td>Gateway</td>
+    <td>9099</td>
+        <td>
+            Gateway WSS
```
Comment (tgtshanika) [inline en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md:303]: WSS / WS - Shall we include web socket/ web socket secure in brackest

File: en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md [hunk 3]
```diff
@@ -91,3 +91,238 @@ In the following diagram, the five components are set up in a distributed deploy
...
+    <th>Port</th>
+    <th>Description</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr class="odd">
+    <td>Key Manager</td>
+    <td>9443</td>
+        <td>
+            Token endpoint and identity rest APIs/web apps
```
Comment (tgtshanika) [inline en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md:275]: REST

File: en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md (hunk 2 above)
Comment (Krishanx92) [reply en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md]: Fixed https://github.com/wso2/docs-apim/pull/2794/commits/53f2b25ff98ad62ce7f3f97ec7273e932e4825b0

File: en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md (hunk 3 above)
Comment (Krishanx92) [reply en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md]: Fixed https://github.com/wso2/docs-apim/pull/2794/commits/53f2b25ff98ad62ce7f3f97ec7273e932e4825b0

**Review by tgtshanika — APPROVED** [review]

**Pull request merged**